code/obj/machinery/computer/card.dm

```diff
@@ -44,6 +44,9 @@
 	throw_speed = 1
 	throw_range = 4
 	w_class = W_CLASS_BULKY
+	stamina_damage = 40
+	stamina_cost = 17
+	stamina_crit_chance = 10
```

Comment:
not sure what's going on with this unlisted buff to briefcases
```suggestion
```

Reply:
Ah sorry for the late response, but this isn't a buff to the briefcases, its just there to keep the unfolder matched to the briefcase, because of how its programed, the unfolder is its own item separate from briefcases.